inst/qml/qml_components/BayesianMetaAnalysisPlots.qml

```diff
@@ -37,58 +37,58 @@ Section
 		CheckBox
 		{
 			id: 		checkForest
-			name: 		"checkForestPlot"
+			name: 		"forestPlot"
```

Comment:
I'm not sure whether that makes any sense, as the changes in selection mechanism and data set structure will make the options incompatible between the last and the next release anyway.

Reply:
So you want to make breaking changes and say that all Cochrane analyses made with jasp < 0.17.0 no longer work in jasp >= 0.17? Then you should add something like [this](https://github.com/jasp-stats/jaspAnova/blob/a192785d513254f03ef804d10b47aaffd3b2f1c1/inst/qml/AnovaRepeatedMeasuresBayesian.qml#L32-L53) to indicate what breaking changes were made to the users. I'd also add a comment about this to the upgrader file though because this will confuse developers in the future (e.g., Note: 0.17.0 introduced breaking changes without a possible upgrade path).

Reply:
yes indeed, I just added the warning message and note to the upgrade file